lib/developer_portal/lib/liquid/drops/api_spec.rb

```diff
@@ -21,6 +21,11 @@ def service
         return unless (service = @spec.service)
         Drops::Service.new(service)
       end
+
+      desc 'Returns the public production host endpoint if it linked to a service `nil` otherwise'
```

Comment:
'Returns the public production host endpoint if **it's** linked to a service**;** `nil` otherwise'